exonum/src/storage/options.rs

```diff
@@ -14,16 +14,25 @@
 
 //! Abstract settings for databases.
 
-/// Options for database.
+/// Options for the database.
+///
+/// These parameters apply to the underlying database of Exonum, currently RocksDB.
 #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct DbOptions {
-    /// Number of open files that can be used by the DB.
+    /// Number of open files that can be used by the database. The underlying
```

Comment:
Please separate the first sentence into a separate para to create a short summary.